src/components/MovieCard/index.tsx

```diff
@@ -22,21 +24,24 @@ const ImageWrapper = styled.View`
 `
 
 interface Props {
-  navigation: NavigationContainerRef
-  movie: Movie | SearchData
+  navigation: any
```

Comment:
You failed to type navigation ?
Why NavigationContainerRef no longer works ?

Reply:
Oh yes, I forgot.
I have to type correctly the navigation with "push" method 👍